src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Microsoft.AspNetCore.Razor.LanguageServer.csproj

```diff
@@ -75,6 +75,7 @@
 
     <RestrictedInternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.Razor" Namespace="Microsoft.AspNetCore.Razor.LanguageServer.Hosting" Key="$(RazorKey)" />
     <RestrictedInternalsVisibleTo Include="rzls" Namespace="Microsoft.AspNetCore.Razor.LanguageServer.Hosting" Key="$(RazorKey)" />
+    <RestrictedInternalsVisibleTo Include="rzls" Namespace="Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts" Key="$(RazorKey)" />
```

Comment:
Please don't expand the RestrictedInternalsVisibleTo to rzls. We're using RestrictedInternalsVisibleTo for MS.ANC.Razor.LanguageServer.Hosting to ensure that we don't end up with any unexpected dependencies outside of the language server. Instead, consider adding an `Endpoints` folder inside of `Hosting` and move the endpoint and its friends there.